crates/utils/src/tracing.rs

```diff
@@ -89,8 +89,8 @@ mod test {
     };
 
     use super::*;
-    #[test]
-    fn test_inject_and_extract() -> Result<(), Box<dyn std::error::Error>> {
+    #[tokio::test]
+    async fn test_inject_and_extract() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Why this needs to be changed to async?

Reply:
That is because `opentelemetry_otlp` require tokio runtime. Without this modification, this test case will result in "there is no reactor running, must be called from the context of a Tokio 1.x runtime".